src/tests/ftest/util/test_utils_container.py

```diff
@@ -488,6 +488,29 @@ def destroy_snap(self, snap_name=None, epc=None, epcrange=None):
         self.daos.container_destroy_snap(**kwargs)
         self.epoch = None
 
+    @fail_on(CommandFailure)
+    def list_snaps(self):
+        """List the snapshots defined for this container.
+
+        Raises:
+            DaosTestError: if the daos container list-snaps command fails
+
+        Returns:
+            list: a list of snapshot epochs defined for this container
+        """
+        self.log.info("Listing snapshots for container: %s", str(self))
+        if not self.daos:
+            raise DaosTestError("Undefined daos command")
+
+        # destroy snapshot using daos utility
+        kwargs = {
+            "pool": self.pool.identifier,
+            "cont": self.identifier,
+        }
+        self._log_method("daos.container_list_snaps", kwargs)
```

Comment:
Do we need these anymore since `container_list_snaps` will log the command string?

Reply:
Done